lib/hacks/gradient.coffee

```diff
@@ -72,7 +72,7 @@ class Gradient extends Value
     if /-?\d+(.\d+)?grad/.test(nodes[0].value)
       nodes[0].value = @normalizeUnit(nodes[0].value, 400)
     else if /-?\d+(.\d+)?rad/.test(nodes[0].value)
-      nodes[0].value = @normalizeUnit(nodes[0].value, 2)
+      nodes[0].value = @normalizeUnit(nodes[0].value, 2 * Math.PI)
```

Comment:
One full circle is 2π radians. **NOT** 2 radians.
https://developer.mozilla.org/en-US/docs/Web/CSS/angle
https://en.wikipedia.org/wiki/Radian